php-checks/src/main/resources/org/sonar/l10n/php.properties

```diff
@@ -20,318 +20,3 @@ rule.php.S1448.param.countNonpublicMethods=Whether or not to include non-public
 rule.php.S1311.param.max=Maximum complexity allowed.
 rule.php.S1200.param.max=Maximum number of classes a single class is allowed to depend upon
 rule.php.S117.param.format=Regular expression used to check the names against.
-rule.php.S1488.name=Local variables should not be declared and then immediately returned or thrown
-rule.php.S125.name=Sections of code should not be "commented out"
-rule.php.S1793.name="elseif" keyword should be used in place of "else if" keywords
-rule.php.S1799.name="exit(...)" and "die(...)" statements should not be used
-rule.php.S1765.name=The "var" keyword should not be used
-rule.php.S1784.name=Method visibility should be explicitly declared
-rule.php.S1781.name=PHP keywords and constants "true", "false", "null" should be in lower case
-rule.php.S1766.name=More than one property should not be declared per statement
-rule.php.S1124.name=Modifiers should be declared in the correct order
-rule.php.S1780.name=Closing tag "?>" should be omitted on files containing only PHP
-rule.php.S1779.name=Only LF character (Unix-like) should be used to end lines
-rule.php.S1788.name=Method arguments with default value should be last
-rule.php.S1808.name=Source code should comply with formatting standards
-rule.php.S1808.param.namespace_blank_line== "namespace" declarations should be followed by a blank line\n\
```

Comment:
@ivandalbosco We should keep parameters' descriptions, just remove code samples from them.
```
rule.php.S1808.param.namespace_blank_line="namespace" declarations should be followed by a blank line
rule.php.S1808.param.use_after_namespace="use" declarations should be placed after "namespace" declarations
...
```